satpy/etc/readers/ahi_hrit.yaml

```diff
@@ -22,7 +22,6 @@ file_types:
     hrit_b01_fd:
         file_reader: !!python/name:satpy.readers.hrit_jma.HRITJMAFileHandler
         file_patterns:
-        - 'IMG_DK{area:02d}B01_{start_time:%Y%m%d%H%M}_{segment:3s}'
```

Comment:
Shouldn't this be used in the `hrit_b01_seg` file type above?

Reply:
Actually yeah this is a big problem. Now there are two file types with the same pattern.

Reply:
That's a typo indeed

Reply:
the one above should have `segment`